pkg/secrets/secrets.go

```diff
@@ -323,14 +332,33 @@ func addFIPSModeSecret(mounts *[]rspec.Mount, containerWorkingDir, mountPrefix,
 
 	if !mountExists(*mounts, secretsDir) {
 		m := rspec.Mount{
-			Source:      filepath.Join(mountPrefix, secretsDir),
+			Source:      ctrDirOnHost,
 			Destination: secretsDir,
 			Type:        "bind",
 			Options:     []string{"bind", "rprivate"},
 		}
 		*mounts = append(*mounts, m)
 	}
 
+	srcBackendDir := "/usr/share/crypto-policies/back-ends/FIPS"
+	destDir := "/etc/crypto-policies/back-ends"
```

Comment:
@rhatdan would it make sense to add the above two variables into containers.conf at some point?

Reply:
Also I'm not a big fan of having a directory defined like this in a middle of a function somewhere, but personal twitch.

Reply:
I don't know if containers.conf is the right place.  The place to extend would be mounts.conf I guess. but we would need some changes to how this is parsed.

Currently 
SRC:DEST

But SRC is on the host and DEST is in the container

This patch basically does SRC in the container to DEST in the container.

Mounts.conf also copies SRC into container local storage and then mounts Container Local Storage at DEST.

The HPC guys want additional volumes "bind" mounted into automatically into the containers.

Reply:
I just added
```
# List of additional volumes. Specified as
# "<directory-on-host>:<derectory-in-container>:<options>", for example: 
# "/db:/var/lib/db:ro".
# If it is empty or commented out, no volumes will be added
# additional_volumes = []
```
To containers.conf, but this does not solve the issue we have here either.